src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptionPages.java

```diff
@@ -206,6 +216,13 @@ public static OptionPage quality() {
                         .setBinding((opts, value) -> opts.quality.enableVignette = value, opts -> opts.quality.enableVignette)
                         .setImpact(OptionImpact.LOW)
                         .build())
+                .add(OptionImpl.createBuilder(boolean.class, vanillaOpts)
+                        .setName(new TranslatableText("options.autosaveIndicator"))
+                        .setTooltip(new TranslatableText("sodium.options.autosave_indicator.tooltip"))
+                        .setControl(TickBoxControl::new)
+                        .setBinding((opts, value) -> opts.showAutosaveIndicator /* Autosave indicator*/ = value, opts -> opts.showAutosaveIndicator)
```

Comment:
Inline comment seems redundant

Reply:
That comment was added back when the autosave indicator field was unmapped, so yes, it is now redundant.